PhenotypeOrthologous.pm

```diff
@@ -0,0 +1,191 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+PhenotypeOrthologous
+
+=head1 SYNOPSIS
+
+ mv PhenotypeOrthologous.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_human.gff3.gz,species=rat
+
+=head1 DESCRIPTION
+
+ A VEP plugin that retrieves phenotype information associated with orthologous genes from from model organisms.
+
+ The plugin presently only supports two model organisms, rats and mouse. 
+
+ The PhenotypeOrthologous file can be downloaded from https://ftp.ensembl.org/pub/current_variation/PhenotypeOrthologous
+
+ The plugin can be run: 
+   
+  ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_mouse_orthologous.gff3.gz
+ 
+  To return only results for rat :
+    ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_mouse_orthologous.gff3.gz,species=rat
+
+    ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_mouse_orthologous.gff3.gz,species=rattus_norvegicus
+  To return only results for mouse:
+    ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_mouse_orthologous.gff3.gz,species=mouse
+
+    ./vep -i variations.vcf --plugin PhenotypeOrthologous,file=rat_mouse_orthologous.gff3.gz,species=mus_muculus
+
+ The tabix utility must be installed in your path to use this plugin.
+ Check https://github.com/samtools/htslib.git for instructions.
+
+=cut 
+
+package PhenotypeOrthologous;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Variation::Utils::BaseVepTabixPlugin;
+
+use base qw(Bio::EnsEMBL::Variation::Utils::BaseVepTabixPlugin);
+
+sub new {
+  my $class = shift;
+
+  my $self = $class->SUPER::new(@_);
+ 
+  $self->expand_left(0);
+  $self->expand_right(0);
+
+  $self->get_user_params();
+    
+  my $params = $self->params_to_hash();
+
+  my $file = $params->{file};  
+  my $species = $params->{species};
+
+  die "File needs to be specified to run the PhenotypeOrthologus plugin. \n" if  (!$file);
+  $self->add_file($file);
+
+  #die "Species needs to be specified to run the PhenotypeOrthologous plugin. \n" if (!$species); 
+
+  if ( defined($species) && $species ne "rattus_norvegicus" && $species ne "rat" && $species ne "mouse" && $species ne "mus_muculus") {
+    die "Phenotype Orthologous plugin only works for rat and mouse \n"
+  }
+  
+  if (defined($species)) {
+    $self->{species} = "rat" if $species eq "rattus_norvegicus" || $species eq "rat";
+    $self->{species} = "mouse" if $species eq "mus_muculus" || $species eq "mouse";
+  }
+
+
+  return $self;
+
+}
+
+sub feature_types {
+  return ['Transcript'];
+}
+
+sub get_header_info {
+  return { PhenotypeOrthologous => 'Phenotypes associated with orthologous genes in model organism'};
```

Comment:
The header could be:
```
PhenotypeOrthologous_MouseOrthologous_geneid
PhenotypeOrthologous_MouseOrthologous_phenotype
PhenotypeOrthologous_RatOrthologous_geneid
PhenotypeOrthologous_RatOrthologous_phenotype
```

Note: if species option is used then the header should only include that species.